doc/build.md

```diff
@@ -45,7 +51,7 @@ The following components are included in the Basketweave keyboard kit:
 
 ### Step 1. Zener Diodes (D69, D70)
 **This part has specific orientation** - black line on the diode lines up with the square pad (points left)
-![Zener Diodes](https://i.imgur.com/sDhJDcs.jpg)
+![Zener Diodes](https://imgur.com/2dWvPp2.png)
```

Comment:
Sorry to be a pain, but the lighting makes the diodes really dark and unclear...